packages/jest-resolve/src/defaultResolver.ts

```diff
@@ -13,28 +13,45 @@ import {
   resolve as resolveExports,
 } from 'resolve.exports';
 import {
-  PkgJson,
+  PackageJson,
   findClosestPackageJson,
   isDirectory,
   isFile,
   readPackageCached,
   realpathSync,
 } from './fileWalkers';
 
-// copy from `resolve`'s types so we don't have their types in our definition
-// files
-interface ResolverOptions {
+type ResolverOptions = {
+  /** Directory to begin resolving from. */
   basedir: string;
   browser?: boolean;
```

Comment:
Can’t get what this option does. Static methods of `Resolver` class simply pass it to resolver module. Perhaps this option should be removed in favour of `conditions: ['browser']`?

For instance, `resolve` package does not have this option. `browser-resolve` has it, but it is of type `string`, not `boolean`. Perhaps it should be passed to `resolve.exports`, because it take `browser?: boolean` option? Hm.. Hard to say.

Reply:
yeah, this is from the old option. kill it with fire!